controller/konnect/ops/ops_errors.go

```diff
@@ -320,3 +322,47 @@ func IgnoreUnrecoverableAPIErr(err error, logger logr.Logger) error {
 
 	return err
 }
+
+func errorIsDataPlaneGroupConflictProposedConfigIsTheSame(err error) bool {
+	var errConflict *sdkkonnecterrs.ConflictError
+	if !errors.As(err, &errConflict) {
+		return false
+	}
+
+	// NOTE: Status contains a float64 value, so we need to cast it to int to deterministically compare.
+	floatStatus, okStatus := (errConflict.Status).(float64)
+	if !okStatus || int(floatStatus) != 409 {
+		return false
+	}
+
+	strDetail, okDetail := errConflict.Detail.(string)
+	if !okDetail ||
+		!strings.Contains(
+			strDetail, "Proposed configuration and current configuration are identical",
```

Comment:
As I understand, unfortunately we need to rely on such descriptive messages that may change in the future, to get it checked